bitcoin/examples/bip32.rs

```diff
@@ -5,7 +5,7 @@ use std::{env, process};
 
 use bitcoin::address::Address;
 use bitcoin::bip32::{ChildNumber, DerivationPath, ExtendedPrivKey, ExtendedPubKey};
-use bitcoin::hashes::hex::FromHex;
+use bitcoin::hex::FromHex;
```

Comment:
I keep going back-and-forth on whether its better to use the re-export within `rust-bitcoin` or use the crate directly? Also we do the same in test modules? This applies to secp also. Would like to get some consensus on this please since there is going to be a lot more of it.

Reply:
- It may cause less merge conflicts if we do not use the re-exported path?
- It may be less maintenance if we do not have to worry about where the module/crate comes from?

Reply:
@sanket1729 I know you told me your opinion in the past but I cannot remember it sorry, can you re-state if you want to please.